packages/react-text/src/components/Text/Text.test.tsx

```diff
@@ -2,9 +2,10 @@ import * as React from 'react';
 import { render } from '@testing-library/react';
 import { Text } from './Text';
 import { isConformant } from '../../common/isConformant';
+import { TextProps } from './Text.types';
```

Comment:
This should probably be:

```suggestion
import type { TextProps } from './Text.types';
```